setup.sh

```diff
@@ -68,10 +68,8 @@ drupal_tests_install() {
   # If this is updated remember to update the Dockerfile too.
   composer require --dev --no-update \
       cweagans/composer-patches \
-      behat/mink-selenium2-driver:1.3.x-dev \
```

Comment:
Removing `behat/mink-selenium2-driver` and `drupal/coder` b/c they are included in `drupal/core-dev`.